src/js/components/awardv2/financialAssistance/FinancialAssistanceContent.jsx

```diff
@@ -73,13 +85,11 @@ export default class FinancialAssistanceContent extends React.Component {
                 </div>
                 <AwardSection type="row">
                     <AwardSection type="column">
-                        <AwardAmountsSection
-                            awardType={this.props.overview.category}
-                            awardOverview={awardAmountData}
-                            tooltipProps={defaultTooltipProps}
-                            jumptoSection={this.props.jumpToSection} />
+                        {amountsSection}
+                    </AwardSection>
+                    <AwardSection type="column">
+                        <FederalAccountsSection />
                     </AwardSection>
-                    <ComingSoonSection title="Description" includeHeader />
```

Comment:
Not sure why we are removing this?

Reply:
I think this was in place of the column for the Federal Accounts section

Reply:
Jk just checked the mockup and this was a placeholder for **Description**. I can add it back and make a new row to hold **Federal Accounts**

Reply:
See comment [here](https://github.com/fedspendingtransparency/usaspending-website/pull/1216#discussion_r321779092) we shouldn't be nesting the `<ComingSoonSection />` in an `<AwardSection type='col' >`